### PR TITLE
Devops/handle ingredient amount to taste etc (PRA-133)

### PR DESCRIPTION
**Pull request by AnnaPestova1**
## One Line Description
[handle ingredientAmount equals -1, -2, -3](https://github.com/Code-the-Dream-School/dd-prac-team1-front/commit/419bdf79b01e302ed5582898722c9a35236bfe4d)
## Requirements

_What this code should do_
in edit recipe you'll be able to choose "to taste", "for serving", "for garnish" options and render it. 
## Notes
errors with saving tags, diets and ingredient amount = 0 handle in pull request  devops/edit-single-recipe
_Any additional info_

## Test Steps

_Instructions on how to test the changes_

## UI/UX Outcome

_Attach a screenshot here_

## Checklist

- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed

### CODE REVIEW

File: src/components/IngredientAmountHandle.tsx
Comment (rebekahcallkacz) [inline src/components/IngredientAmountHandle.tsx:1]: I might name this something more representative of what's displayed on the page. For example, `IngredientAmountSelect`

File: src/components/EditRecipe.tsx [hunk 1]
```diff
@@ -0,0 +1,655 @@
...
+          isClosable: true,
+          position: "top"
+        });
+      });
+  };
+
+  if (recipe === null) return null;
+  if (ingredients === null) return null;
+  if (tags === null) return null;
+  if (diets === null) return null;
```
Comment (rebekahcallkacz) [inline src/components/EditRecipe.tsx:88]: What is the purpose of these lines?

Comment (rebekahcallkacz) [reply src/components/EditRecipe.tsx:88]: Usually, all of your return statements for a functional component should be at the bottom to make it easier to read.

**Review by rebekahcallkacz — APPROVED** [review]
Based on the name of the branch/PR, I only tested the different ingredient options which looks good. Let me know if there was more I was supposed to test in this PR.

EDIT: Just saw your message about not checking this one. Feel free to use or ignore my feedback!

**Comment by AnnaPestova1** [comment]
all code from here with fixed bugs in pull request #41 Devops/edit image in edit recipe (PRA-123) (PRA-136) (PRA-133)

**Pull request closed without merging**